File: PSP/borradores/ae/jwt/AlumnoSimulacro/src/main/java/org/example/alumnosimulacro/domain/model/Alumno.java
package org.example.alumnosimulacro.domain.model;

import com.fasterxml.jackson.annotation.JsonTypeInfo;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;
@Data
@AllArgsConstructor
@Builder
@NoArgsConstructor
public class Alumno {
    private String nombre;
    private int id;
    List<Asignatura> asignaturas;
}

File: MOBILES/simulacros/primerexamen/FinalExamenMoviles_AdrianSaavedra/app/src/main/java/com/example/finalexamenmoviles_adriansaavedra/domain/model/Alumno.kt
package com.example.finalexamenmoviles_adriansaavedra.domain.model


data class Alumno(
val nombre : String = "",
val apellido : String = "",
val dni : String = "",
val email : String = "",
val asignaturas : List<Asignatura>? = emptyList()
)